src/credentials/examples/GroupDataProviderExample.cpp

```diff
@@ -382,7 +387,13 @@ class StaticGroupsProvider : public GroupDataProvider
             {
                 entry.group    = mapping.group;
                 entry.endpoint = mapping.endpoint;
-                entry.in_use   = true;
+                if (nullptr != mapping.name.data())
+                {
+                    entry.nameLength = std::min(mapping.name.size(), kGroupNameMax) + 1;
+                    strncpy(entry.name, mapping.name.data(), entry.nameLength);
+                    entry.name[entry.nameLength] = 0;
+                }
```

Comment:
```suggestion
                }
                else
                {
                    entry.name[0] = 0;
                }
```
or some such?  Otherwise it seems like we would end up with un-initialized data there.

Reply:
Fixed.